README.md

```diff
@@ -5,28 +5,40 @@
-## App Connector Client
-App connector has a REST api that is used to get certificate from kyma cluster. You can run it with `npm start` or you can use the *Dockerfile* to create a docker image. Further instructions are in [app-connector-client/README.md](app-connector-client/README.md).
+## Overview
 
-## OpenAPI Mock
-Mocks REST calls given an OpenAPI specification. [openapi-mock/README.md](openapi-mock/README.md).
+Varkes (Greek for “small boats”) is a framework which allows you to develop your functionality in an easy way, even if it’s based on a heavy application. With this small boat at hand, you can mock the heavy application (the “big ship”) within minutes to start developing your functionality. Varkes allows you to experiment with Kyma and develop your applications without thinking about their deployment. 
 
-## Odata Mock
-Mocks rest calls given an OData specification [odata-mock/README.md](odata-mock/README.md).
+
+Varkes framework provides the following npm components to make the application mocks work:
+
+* Application Connector Client, which is used to get the certificate from the Kyma cluster. You can run it with `npm start` or use the Dockerfile to create a docker image. For details, see [Application Connector Client](app-connector-client/README.md).
+
+* OpenAPI mock, which mocks the application APIs based on the OpenAPI specification. For details, see [OpenAPI mock](openapi-mock/README.md).
+
+* Odata Mock, which mocks the application APIs based on the OData specification. For details, see [OData mock](odata-mock/README.md).
+
+## Architecture
+
+The diagram shows you how an application mock integrates with a Kyma cluster and which functionality is taken over by which framework modules:
+
+![Mocks Architecture](/assets/mocks-architecture.svg)
 
 ## Examples
-Examples folder provides mocks written using openapi & odata packages. If you want to use them without cloning the whole repo, please refer to tags in `release` branch. There, you can find the latest version of the dependencies already included in their package.json files. Working on examples in master branch without cloning the whole repo is currently impossible because they use development version numbers in their package.jsons and these are not published to npm registry.
 
-To create docker images of examples, please also refer to `release` branch, where npm installed in docker containers can get the latest version from the registry.
+The `examples` folder provides mocks written using OpenAPI and OData packages. 

```

Comment:
I think we can have a very small readme.md file for the Examples folder as well. We can then have just a link here.